app.js

```diff
@@ -171,3 +171,5 @@ app.use(function (err, req, res, next) {
   res.status(err.status || 500)
   res.render('http-error', { error: 'Internal server error', message: err })
 })
+
+module.exports = server
```

Comment:
Is `server` used in `app.spec.js`? It looks like it might be defined again in that file.

Reply:
We're exporting the server and import it in the spec as `app` on the following line
`const app = require('../app.js')`

we then save the reference in a local variable and start the server:
`server = app.listen(3000)`

and we're doing that to be able to stop the server after finishing the tests:
`server.close()`

Reply:
Cool that makes sense. Thanks for explaining 👍